lib/i18n_routable/config.rb

```diff
@@ -5,6 +5,7 @@ module Config
     self.localize_config = {}
 
     def setup! options
+      I18n.config.enforce_available_locales = false
```

Comment:
http://stackoverflow.com/questions/20361428/rails-i18n-validation-deprecation-warning

The errors are raised whenever the following methods are called:
```I18n.config.default_locale=
I18n.config.locale=
I18n.translate
I18n.localize
I18n.transliterate
```
So basically, every test breaks.

> Before this change, if you passed an unsupported locale, Rails would silently switch to it if the locale is valid (i.e. if there is a corresponding locale file in the /config/locales folder), otherwise the locale would default to the config.i18n.default_locale configuration (which defaults to :en).

So we have to set it to false to resort to the old behaviour. Otherwise, we have to make sure I18n.available_locales is explicitly set. I tried to implement this, but it got a little messy with the way some of the tests work. For example, spec_routes.rb does `:locales => [{:gibberish => :gibb}, :aussie, :es, :fr, :'fr-CA']`, which, if we set as the available_locales, complains because {:gibberish => :gibb} can't be converted to a symbol ([source](https://github.com/svenfuchs/i18n/blob/f19893d84262261d9c8abe303f465ea28ba057e4/lib/i18n/config.rb#L55)).